addusers.py

```diff
@@ -285,7 +256,13 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
-    openstack = Openstack(admin_user, admin_pwd, admin_tenant, auth_url, nova_version)
+    auth = v3.Password(auth_url=auth_url,
+                       username=admin_user,
+                       user_domain_id = 'default',
+                       password=admin_pwd)
```

Comment:
I got this part from Kristi's instructions for Setpass.  I think maybe a normal user would need to specify a project, but because we are operating as an admin and the project is specified in the operations we do, we do not need to specify a project to authenticate.  But, I'm not 100% sure about that.

In my test environment (and our production environment) the script doesn't authenticate directly as the 'admin' user, there's a separate user which has the role 'admin' in the services project, and role '_member_' in the admin project.   Which seems to work OK without specifying a project.